spec/order_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'order'
+
+describe Order do 
+
+	let(:menu) {double :menu, dishes: [curry,kebab]}
+	let(:curry) {double :dish, name: "curry", price: 500}
+	let(:kebab){double :dish, name: "kebab", price: 450}
+	let(:order) {Order.new}
+
+	it 'should be able to store a list of dishes' do
+		expect(order.total_item_count).to eq (0)
+	end
+
+	it 'should be able to select items from the menu and add them to the list' do
+		order.add_item(menu,curry)
+		order.add_item(menu,kebab)
+		expect(order.total_item_count).to eq (2)
+		expect(order.order_items).to eq [curry,kebab]
+	end
+
+	it 'should be able to total the cost of one item in the order list' do
+		order.add_item(menu,curry)
+		expect(curry).to receive(:price)
```

Comment:
I think it would make more sense to not specify the price in the double above and specifiy the return value here like expect(curry).to receive(:price).and_return(500)